code/modules/cooking/recipes/cultural/recipes_tajara.dm

```diff
@@ -71,6 +71,90 @@
 			return ..()
 	return FALSE
 
+// Tajaran cakes
+/decl/recipe/conecake
+	appliance = OVEN
+	reagents = list(/decl/reagent/drink/milk/adhomai = 5, /decl/reagent/nutriment/flour/nfrihi = 15, /decl/reagent/water = 5, /decl/reagent/sugar = 15)
+	result = /obj/item/reagent_containers/food/snacks/cone_cake
+
+/decl/recipe/avah
+	appliance = FRYER
+	reagents = list(/decl/reagent/drink/milk/adhomai = 5, /decl/reagent/sugar = 5)
+	items = list(
+		/obj/item/reagent_containers/food/snacks/dough,
+		/obj/item/reagent_containers/food/snacks/cheesewedge
+	)
+	result = /obj/item/reagent_containers/food/snacks/avah
+
+// Tajaran pies
+/decl/recipe/fruitRikazu
```

Comment:
Small stylistic quirk, can you make it consistent with the other rikazu paths below this (lowercase R)?

Reply:
My bad, I thought I got them all.